opentelemetry-api/src/opentelemetry/metrics/aggregation.py

```diff
@@ -11,3 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+#TODO
+class Aggregation:
```

Comment:
I'm surprised to see aggregations in the API, but don't have a solid argument for either package. Why did you put them here?

Reply:
Aggregations are not mentioned in the current RFC except to describe the default interpretation.  Please review https://github.com/open-telemetry/oteps/pull/29